README.md

```diff
@@ -90,6 +165,52 @@ provided as repository secrets.
 
 [This action](https://github.com/styfle/cancel-workflow-action) is a replacement for the Travis settings **Auto cancel branch builds** and **Auto cancel pull request builds**.
 
+### Triggering a workflow in another repository
+
+[actions/github-script](https://github.com/actions/github-script) can be used, here is a sample:
+
+```yml
+      - name: Trigger Downstream Builds
+        if: steps.is_default_branch.outputs.result == 'true'
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.BOT_GITHUB_TOKEN }}
+          script: |
+            await github.rest.actions.createWorkflowDispatch({
+              owner: 'Alfresco',
+              repo: 'alfresco-process-connector-services',
+              workflow_id: 'build.yml',
+              ref: 'develop'
+            });
+```
+
+Note that this requires using a dedicated token.
+
+Also, the triggered workflow should allow workflow dispatch in its definition (and this configuration should be setup
+on the default branch):
+
+```yml
+on:
+  # allows triggering workflow manually or from other jobs
+  workflow_dispatch:
+```
+
+### rtCamp/action-slack-notif
```

Comment:
shall we suggest this instead of the official Slack action? https://github.com/slackapi/slack-github-action

Reply:
:+1: this should be updated when tackling https://alfresco.atlassian.net/browse/AAE-9076